html/network.html

```diff
@@ -55,6 +60,34 @@ <h5 class="card-title"><strong>Output</strong></h5>
         <script src="/static/js/jquery-3.3.1.slim.min.js"></script>
         <script src="/static/js/bootstrap.min.js"></script>
         <script>
+          function handleEnableChange(event) {
+            event.preventDefault();
+            var checkBox = event.target;
+            var value = checkBox.checked;
+            fetch("/online-state", {
+              method: "POST",
+              headers: {
+                "Content-Type": "application/json",
+              },
+              body: JSON.stringify({online: value})
+            })
+              .then(function(response) {
+                response.json().then(function(json) {
+                  checkBox.checked = json["state"];
+                  var message = document.getElementById("resultMessage")
```

Comment:
because jQuery is available (used by Bootstrap) this can be done more concisely like this:
```js
var message = $('resultMessage');
```

No biggie either way, just FYI